qunicorn_core/db/models/device.py

```diff
@@ -26,7 +26,10 @@ class DeviceDataclass(DbModel):
     """Dataclass for storing CloudDevices of a provider
 
     Attributes:
+        url (str): Rest-endpoint how to connect to the Cloud device
```

Comment:
Similar to comment on 'deployment.py' -> should we reorder the whole models an put mandatory and useful things like 'id' at the top?